### PR TITLE
Option audio focus

### PR DESCRIPTION
**Pull request by wemped**
I noticed that ToroVideoView always requests the AudioFocus of the device.

This is a simplistic approach to allow background audio.

### CODE REVIEW

**Comment by eneim** [comment]
Hi there, thanks for your PR. I'm AKF for now. Please wait for the review, I'll check out ASAP.

File: toro/src/main/java/im/ene/lab/toro/widget/ToroVideoView.java [hunk 1]
```diff
@@ -614,7 +615,9 @@ private void openVideo() {
     release(false);
 
     AudioManager am = (AudioManager) getContext().getSystemService(Context.AUDIO_SERVICE);
-    am.requestAudioFocus(null, AudioManager.STREAM_MUSIC, AudioManager.AUDIOFOCUS_GAIN);
+    if (mRequestAudioFocus){
```
Comment (eneim) [inline toro/src/main/java/im/ene/lab/toro/widget/ToroVideoView.java]: Really tiny fix: please add one space between ```){`` (here and else where). Please keep the code format too.

**Comment by eneim** [comment]
No problem found. I understand that you would like to allow this VideoView to have background playback. This sounds non-sense for Toro since it will play Video only on foreground. But for a wider usage, I support this PR. Please review my comment, make the fix so I could merge it. Thanks.

**Comment by wemped** [comment]
Updated according to comments

**Comment by eneim** [comment]
Thanks @wemped , I would like to wait for response from another issue which may relates to this PR in one day then I will merge this.

**Pull request merged**